cypress/e2e/help-section/helpAreas.feature

```diff
@@ -3,45 +3,32 @@
 
-  As a teacher I want to use the help areas in the header and sidebar so that I can find help when needed
+  As a user I want to use the help areas in the header and sidebar so that I can find help when needed
 
-  Scenario Outline: teacher can use the help area in the (header, sidebar), search article and submit an issue via contact form
-    Given I am logged in as a '<teacher>' at '<namespace>'
-    When I click on the question icon in header
-    And I click on help section in header
-    Then I can see the help articles page
-    When I click on the question icon in header
-    And I click on send request or problem in header
-    Then I can see the help contact page
-    When I click on the question icon in header
-    And I click on advanced trainings in header
-    #Then a new tab in browser opens
-
-    # use the help area in the sidebar
-    When I go to help section in sidebar
-    Then I can see the help articles page
-    When I go to help articles in sidebar
-    Then I can see the help articles page
-    When I go to contact in sidebar
-    Then I can see the help contact page
-    When I go to advanced trainings in sidebar
+  Scenario Outline: User can use the help area in the (header, sidebar), search article and submit an issue via contact form
+    Given I am logged in as a '<user>' at '<namespace>'
+    When I click on Help Section in sidebar
+    When I click on Help articles in sidebar
+    Then I see the help articles page
+    When I click on Contact in sidebar
+    Then I see the help contact page
+    When I click on  Advanced trainings in sidebar
     #Then a new tab in browser opens
 
     # use the article search inside the help articles area
-    When I go to help section in sidebar
-    Then I can see the help articles page
-    When I enter keyword in search bar
-    Then I can see an help article related to my search
+    When I click on Help articles in sidebar
+    When I enter '<search_term>' in search bar for help articles
+    Then I see an help article containing '<search_result>'
 
     # submit an issue via contact form inside help area
-    When I go to help section in sidebar
-    When I go to contact in sidebar
-    Then I can see the help contact page
-    When I fill out the contact form
-    Then I can send it to the support
+    When I click on Contact in sidebar
+    Then I see the help contact page
+    When I fill out the contact form with option '<contact_option>', subject '<contact_subject>' and sender email '<contact_email>'
+    When I click on button Submit to send form
+    Then I see message 'Feedback erfolgreich versendet!'
 
     @school_api_test
     @staging_test
     Examples:
-      | teacher      | namespace |
-      | teacher1_brb | brb       |
\ No newline at end of file
+      | user         | namespace | search_term      | search_result               | contact_option | contact_subject                     | contact_email    |
```

Comment:
Here column name should be the 'role name' instead of 'user'?

Example:
| namespace | admin          | teacher          | 
| brb             | admin1_brb | teacher1_brb |

Reply:
I would keep it as user, because then you can extend the test with different examples and execute it with different roles.

Reply:
clarified